include/crm/msg_xml.h

```diff
@@ -36,8 +36,11 @@ extern "C" {
 /* This has been deprecated as a CIB element (an alias for <clone> with
  * "promotable" set to "true") since 2.0.0.
  */
-#define PCMK_XE_PROMOTABLE_LEGACY           "master"
+#  define PCMK_XE_PROMOTABLE_LEGACY         "master"
 
+#  define PCMK_XE_DATE_EXPRESSION           "date_expression"
+#  define PCMK_XE_OP_EXPRESSION             "op_expression"
+#  define PCMK_XE_RSC_EXPRESSION            "rsc_expression"
```

Comment:
I've never been thrilled with indenting defines; I think it takes up space and looks weird with more than one level of indenting, without particularly helping readability. Also these should in alphabetic order for ease of maintenance once everything is moved here.

Reply:
I've never indented defines in my own code unless it's maybe in an immediate block.

I was keeping them in "logical" blocks (arbitrary) but I'm happy with alphabetical. Eliminates the need to spend time deciding "where does this new one go?"